src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -20,13 +24,15 @@ public abstract class EditCommand extends Command {
             + "by the index number used in the displayed client list. "
             + "Existing values will be overwritten by the input values.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + PREFIX_MODE + "MODE (must be 'client' or 'transaction' or 'remark') "
-            + "[" + PREFIX_NAME + "NAME] "
-            + "[" + PREFIX_ADDRESS + "ADDRESS] "
-            + "[" + PREFIX_PHONE + "PHONE] "
-            + "[" + PREFIX_EMAIL + "EMAIL] "
-            + "[" + PREFIX_TAG + "TAG]...\n"
-            + "Example: " + COMMAND_WORD + " 1 " + PREFIX_MODE + "client " + PREFIX_ADDRESS + "Blk 221 Yishun St 81";
-
+            + "[" + PREFIX_MODE + "client/transaction/remark] "
```

Comment:
Should the `[ ]` be removed? Since `[ ]` denotes that the line is optional however, `PREFIX_MODE` is required.

Reply:
Thanks! They have been removed